WordPress/Classes/ViewRelated/Jetpack/Branding/JetpackBrandingVisibility.swift

```diff
@@ -1,30 +1,28 @@
-
-import Foundation
-
```

Comment:
Big pet peeve of mine: Just because the Xcode template for a new file imports Foundation, it doesn't mean we need to keep it if we don't use Foundation in the file. I don't know if it makes any difference at compile time – I doubt it would – but it just seem like cruft.